src/user/dto/loginUser.dto.ts

```diff
@@ -0,0 +1,12 @@
+import {IsNotEmpty, MinLength, MaxLength} from 'class-validator';
+
+export class LoginUserDto {
+    @IsNotEmpty()
+    @MinLength(6)
```

Comment:
Missing type check at least `@IsString()`
Also, max length for login might be good idea